### PR TITLE
Added SQA documentation for the rDG module

### PR DESCRIPTION
**Pull request by joshuahansel**
Refs #15968

### CODE REVIEW

File: modules/rdg/doc/content/sqa/rdg_sdd.md [hunk 1]
```diff
@@ -1 +1,58 @@
...
+
+!style halign=left
+The MooseApp is the top-level object used to hold all of the other objects in a simulation. In a
+normal simulation a single MooseApp object is created and "run()". This object uses its Factory
+objects to build user-defined objects which are stored in a series of Warehouse objects and
+executed. The Finite Element data is stored in the Systems and Assembly object while the domain
+information (the Mesh) is stored in the Mesh object. A series of threaded loops are used to run
+parallel calculations on the objects created and stored within the warehouses.
+
+MOOSE's pluggable systems are documented on [MOOSE website](https://mooseframework.inl.gov), and those
```
Comment (cticenhour) [inline modules/rdg/doc/content/sqa/rdg_sdd.md:52]:
```suggestion
MOOSE's pluggable systems are documented on the [MOOSE website](https://mooseframework.inl.gov), and those
```

**Review by cticenhour — CHANGES_REQUESTED** [review]
One typo. Otherwise looks good as far as markdown.

**Review by lindsayad — APPROVED** [review]
It's funny that this language describes exactly what we've reproduced in the FV portion of the `navier_stokes` module

**Comment by lindsayad** [comment]
@joshuahansel does anyone use the rDG module that you know of?

**Comment by joshuahansel** [comment]
> @joshuahansel does anyone use the rDG module that you know of?

I think THM may use one class from it, and then Bighorn. That's what I know of. It'd be great if it could be dissolved, if that's what you're getting at...

**Review by cticenhour — APPROVED** [review]

**Comment by cticenhour** [comment]
Griffin failure related to GitHub Enterprise migration. OK to merge.

**Pull request merged**